deckhouse-controller/pkg/apis/deckhouse.io/validation/register.go

```diff
@@ -39,14 +39,9 @@ type moduleManager interface {
 }
 
 // RegisterAdmissionHandlers registers validation webhook handlers for admission server built-in in addon-operator
-func RegisterAdmissionHandlers(
-	registerer registerer,
-	client client.Client,
-	mm moduleManager,
-	configValidator *configtools.Validator,
-	storage moduleStorage,
-	metricStorage *metricstorage.MetricStorage) {
-	registerer.RegisterHandler("/validate/v1alpha1/module-configs", moduleConfigValidationHandler(client, storage, metricStorage, configValidator))
-	registerer.RegisterHandler("/validate/v1alpha1/modules", moduleValidationHandler())
-	registerer.RegisterHandler("/validate/v1/configuration-secret", kubernetesVersionHandler(mm))
```

Comment:
why? saw some PRs flattening arguments and vice versa, let's agree on one form or another.

Reply:
I can't decide, I don't like long lines, but I also don't like parameters in multiple lines. Decide for me :)